Makefile

```diff
@@ -51,6 +51,9 @@ install: bin ## Install binaries to GOPATH
 api-dev: ## Server and watch gitops-server, will reload automatically on change
 	reflex -r '.go' -R 'node_modules' -s -- sh -c 'go run -ldflags $(LDFLAGS) cmd/gitops-server/main.go'
 
+cluster-dev: ## Start tilt to do development with wego-app running on the cluster
+	tilt up
```

Comment:
Could we extend this to also run the cluster script in tools? So I can run a single command to get the cluster up then start tilt. :) Just a suggestion though.